x-pack/test/api_integration/apis/cloud_security_posture/status/status_not_deployed.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import type { CspSetupStatus } from '@kbn/cloud-security-posture-plugin/common/types';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { createPackagePolicy } from '../helper';
+
+export default function (providerContext: FtrProviderContext) {
+  const { getService } = providerContext;
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('GET /internal/cloud_security_posture/status', () => {
+    let agentPolicyId: string;
+
+    describe('STATUS = NOT-DEPLOYED and STATUS = NOT-INSTALLED TEST', () => {
```

Comment:
just wondering why this file is mixing both requirements, i think everything was well organized and was very easy to read and follow and personally i would separate there 2 cases just to stick with the structure. but its your call, it doesn't effect anything. 

maybe consider changing filename to `status_not_installed_not_deployed` or something that indicates that both cases are here